src/components/Identity/processIdSyncs.js

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { assign, cookie, fireDestinations } from "../../utils";
+import namespace from "../../constants/namespace";
+
+const millisecondsPerHour = 60 * 60 * 1000;
+
+// TODO: use alloy cookie once https://github.com/adobe/alloy/pull/26 is merged
+const getControlObject = () => {
+  const obj = {};
+  const val = cookie.get(`${namespace}idSyncControl`) || "";
+  let arr = [];
+
+  if (val) {
+    arr = val.split("_");
+  }
+
+  arr.forEach(pair => {
+    const [id, ts] = pair.split("-");
+
+    obj[id] = ts;
+  });
+
+  return obj;
+};
+
+const setControlObject = obj => {
+  const arr = [];
+
+  Object.keys(obj).forEach(id => arr.push(`${id}-${obj[id]}`));
+
+  cookie.set(`${namespace}idSyncControl`, arr.join("_"), {
+    expires: 6 * 30 // 6 months
+  });
+};
+
+export default ({ destinations, config, logger }) => {
+  const { idSyncsEnabled = true } = config;
```

Comment:
Let's add `idSyncsEnabled` to the configuration map in the `Identity Component` and default it there to `true` instead of doing it here.